### PR TITLE
zh-translation:/docs/reference/config/security/conditions/index.md

### PR DESCRIPTION
**Pull request by ghl1024**
ref: [servicemesher/istio-official-translation/issues/1493](https://github.com/servicemesher/istio-official-translation/issues/1493)

### CODE REVIEW

**Comment by ghl1024** [comment]
@tanjunchen
麻烦闲了帮我review下，谢谢。

File: content/zh/docs/reference/config/security/conditions/index.md [hunk 1]
```diff
@@ -1,38 +1,36 @@
...
+| `request.headers` | `HTTP` 请求头，需要用 `[]` 括起来 | HTTP only | `key: request.headers[User-Agent]`<br/>`values: ["Mozilla/*"]` |
+| `source.ip`  | 源 `IP` 地址，支持单个 `IP` 或 `CIDR` | HTTP and TCP | `key: source.ip`<br/>`values: ["10.1.2.3"]` |
+| `source.namespace`  | 源负载实例命名空间 | HTTP and TCP | `key: source.namespace`<br/>`values: ["default"]` |
+| `source.principal` | 源负载的标识 | HTTP and TCP | `key: source.principal`<br/>`values: ["cluster.local/ns/default/sa/productpage"]` |
+| `request.auth.principal` | 已认证过 `principal` 的请求。 | HTTP only | `key: request.auth.principal`<br/>`values: ["accounts.my-svc.com/104958560606"]` |
+| `request.auth.audiences` | 此身份验证信息的目标主体 | HTTP only | `key: request.auth.audiences`<br/>`values: ["my-svc.com"]` |
+| `request.auth.presenter` | 证书的颁发者 | HTTP only | `key: request.auth.presenter`<br/>`values: ["123456789012.my-svc.com"]` |
+| `request.auth.claims` | `Claims` 来源于 `JWT`。需要用 `[]` 括起来 | HTTP only | `key: request.auth.claims[iss]`<br/>`values: ["*@foo.com"]` |
+| `destination.ip` | 目标IP地址，支持单个 `IP` 或 `CIDR` | HTTP and TCP | `key: destination.ip`<br/>`values: ["10.1.2.3", "10.2.0.0/16"]` |
```
Comment (tanjunchen) [inline content/zh/docs/reference/config/security/conditions/index.md]: 目标 IP 地址

File: content/zh/docs/reference/config/security/conditions/index.md [hunk 2]
```diff
@@ -1,38 +1,36 @@
...
+| `destination.port` | 目标IP地址上的端口，必须在 `[0，65535]` 范围内 | HTTP and TCP | `key: destination.port`<br/>`values: ["80", "443"]` |
```
Comment (tanjunchen) [inline content/zh/docs/reference/config/security/conditions/index.md]: 目标 IP 地址

**Comment by ghl1024** [comment]
@tanjunchen

File: content/zh/docs/reference/config/security/conditions/index.md [hunk 3]
```diff
@@ -1,38 +1,36 @@
...
+title: 授权策略
+description: 授权策略中支持的条件。
 weight: 30
 aliases:
     - /zh/docs/reference/config/security/conditions/
 ---
 
-This page describes the supported keys and value formats you can use as conditions
-in the `when` field of [authorization policy resources](/zh/docs/reference/config/security/authorization-policy/).
+此页面描述了可以用作[授权策略](/zh/docs/reference/config/security/authorization-policy/) `when` 字段中受支持的键和值的格式。
```
Comment (tanjunchen) [inline content/zh/docs/reference/config/security/conditions/index.md]: 受->所

File: content/zh/docs/reference/config/security/conditions/index.md [hunk 4]
```diff
@@ -1,38 +1,36 @@
...
 {{< /warning >}}
 
-For more information, refer to the [authorization concept page](/zh/docs/concepts/security/#authorization).
+有关更多信息，请参阅[授权概念页面](/zh/docs/concepts/security/#authorization)。
 
-## Supported Conditions
+## 支持条件{#supported-conditions}
 
-| Name | Description | Supported Protocols | Example |
+| 名称 | 描述 | 支持的协议 | 例子 |
```
Comment (tanjunchen) [inline content/zh/docs/reference/config/security/conditions/index.md]: 例子 -> 示例

**Pull request merged**